core-bundle/src/EventListener/RequestTokenListener.php

```diff
@@ -73,6 +76,21 @@ public function __invoke(RequestEvent $event): void
         throw new InvalidRequestTokenException('Invalid CSRF token. Please reload the page and try again.');
     }
 
+    public static function isSimpleCorsRequest(Request $request): bool
+    {
+        $contentType = HeaderUtils::split($request->headers->get('content-type'), ';')[0] ?? '';
+
+        return \in_array(
+            $contentType,
```

Comment:
```suggestion
            strtolower($contentType),
```
According to https://datatracker.ietf.org/doc/html/rfc7231#section-3.1.1.1 the content type is case-insensitive